enhancements/machine-api/out-of-tree-provider-support.md

```diff
@@ -0,0 +1,182 @@
+---
+title: neat-enhancement-idea
+authors:
+  - "@Danil-Grigorev"
+reviewers:
+  - "@enxebre"
+  - "@elmiko"
+  - "@mfedosin"
+approvers:
+  - "@enxebre"
+  - "@elmiko"
+  - TBD
+creation-date: 2020-08-31
+last-updated: 2020-09-01
+status: implementable
```

Comment:
Given Draft / WIP status, provisional is probably more appropriate?

Reply:
It seems it is closer to the implementable phase now :)